prepare-embargoed-branches.sh

```diff
@@ -47,6 +51,18 @@ else
 	test -n "$previous_version_prefix" || die "Invalid version: '$version'"
 	branch_name=mingit-$previous_version_prefix.x-releases
 fi
+case "$previous_version" in
+2.46.3)
+	# There was no Git for Windows v2.46.3, so we use v2.46.2
+	previous_version=2.46.2
+	previous_version_prefix=2.46.2
+	;;
+2.47.2)
+	# There was no Git for Windows v2.47.3, so we use v2.47.1(2)
```

Comment:
/2.47.3/2.47.2/

Reply:
Actually, no... This `switch` case is about the _previous_ version, that is, the script is called with `v2.47.3` and `previous_version` will then be set to `2.47.2`...

Reply:
Yes, the switch seems to be correct, but the comment claims that 2.47.3 didn't exist, whereas the code special cases 2.47.2 not existing.